src/github_runner_image_builder/openstack_builder.py

```diff
@@ -246,17 +251,25 @@ def run(
         runner_version=image_config.runner_version,
         proxy=cloud_config.proxy,
     )
+    builder_name = _get_builder_name(
+        arch=image_config.arch, base=image_config.base, prefix=cloud_config.prefix
+    )
+    builder_key_name = _get_keyname(prefix=cloud_config.prefix)
     with openstack.connect(cloud=cloud_config.cloud_name) as conn:
+        _ensure_openstack_resources(
```

Comment:
perhaps rename to `_prepare_openstack_resources`

Reply:
Updated, thank you